src/fixit/__init__.py

```diff
@@ -11,10 +11,23 @@
 
 from .api import fixit_bytes, fixit_file, fixit_paths
 from .rule import LintRule
-from .rule.cst import CSTLintRule, CstLintRule, InvalidTestCase, ValidTestCase
-from .types import Config, FileContent, Result
-
+from .rule.cst import CSTLintRule, CstLintRule
+from .testing import InvalidTestCase, ValidTestCase
+from .types import CodePosition, CodeRange, Config, FileContent, Result
 
 __all__ = [
     "__version__",
+    "fixit_bytes",
```

Comment:
I understand this is for convenience, but when determining backwards compatibility is your intent to look at .api or this`__all__`?

Reply:
I suppose we could look at both, but maybe it makes sense to consider this `__all__` as the public api. Do you have opinions?

Reply:
I'm in favor of solidifying `__all__` as the public API, and having that be the list of things documented in an API reference. I would expect that any data types that we would think users will need when writing lint rules should be added to this list, so that there's a single location to import from.

Reply:
Sounds fine. I've been waffling on this myself in other projects, mainly due to doing expensive imports up front if not needed for all parts of the api. I don't think that applies here.